# HG changeset 9711 patch
# User kaf24@firebug.cl.cam.ac.uk
# Node ID ef1a49d180aca500c85969755f80235ccde6eb37
# Parent  28ffbefb0966106b1a79eedd733c0d5ec9883480
Use halt() instead of asm("hlt").
From: Jan Beulich <jbeulich@novell.com>
References: 177511
xen-unstable changeset:   10259:8425df2203c9b0b7eac8c7061801c643133b8c9b
xen-unstable date:        Fri Jun  2 12:14:27 2006 +0100

Index: head-2006-06-06/arch/x86_64/kernel/smp-xen.c
===================================================================
--- head-2006-06-06.orig/arch/x86_64/kernel/smp-xen.c	2006-06-06 12:51:13.000000000 +0200
+++ head-2006-06-06/arch/x86_64/kernel/smp-xen.c	2006-06-06 12:52:23.000000000 +0200
@@ -488,7 +488,7 @@ static void smp_really_stop_cpu(void *du
 {
 	smp_stop_cpu(); 
 	for (;;) 
-		asm("hlt"); 
+		halt();
 } 
 
 void smp_send_stop(void)
